gobblin-admin/src/main/java/gobblin/cli/CliTablePrinter.java

```diff
@@ -0,0 +1,194 @@
+/* (c) 2015 NerdWallet All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.cli;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.base.Preconditions;
+import com.google.common.primitives.Ints;
+
+
+/**
+ * A format helper for CLI output. Unfortunately it only supports strings, so
+ * values need to be converted previous to passing in. This is done in order to
+ * support table-like formatting.
+ * <p/>
+ * It's recommended that this class is built using the inner {@link Builder} class.
+ *
+ * @author ahollenbach@nerdwallet.com
+ */
+public class CliTablePrinter {
+  /**
+   * Labels for each columns
+   */
+  private List<String> labels;
+
+  /**
+   * A list of flag strings (corresponding to each column)
+   */
+  private List<String> flags;
```

Comment:
Can you explain a bit more what flags mean? I see now they are the flags used for String formatting, but it wasn't clear at the beginning.